NEWS.md

```diff
@@ -1,6 +1,7 @@
 ### Unreleased
 
 * Fixed object labels to adjust to application font changes
+* Tweaked handling of Hexagonal (Staggered) maps by Terrain Brush (#3617)
```

Comment:
I think this wording might mislead people who only read the changelog into thinking Terrains actually work with hex maps. I think it would be good to make it clear here that this does not add general hexagonal Terrain support, since the only immediately useful feature of this change is that side length 0 maps work nicely.

Reply:
Yeah, I thought the same thing, but could not think of a wording that would make that clear, which would not be overly verbose. It might be better to just leave out the changelog entry?

Reply:
I think it's best to include changes that affect behaviour, even if they're not meaningful to most people.
Maybe like this?
> Improved Terrain Brush for isometric-like Hexagonal (Staggered) maps with side length 0.

Reply:
I've left out the "isometric-like" since I think it's still too verbose. I hope adding "with side length 0" is specific enough, heh.

Reply:
I included it in there just to emphasise how much of an edge-case this is xP